pkg/models/business_case.go

```diff
@@ -82,67 +90,79 @@ func (e *EstimatedLifecycleCosts) Scan(src interface{}) error {
+	AlternativeAHostingCloudServiceType null.String        `json:"alternativeAHostingCloudServiceType" db:"alternative_a_hosting_cloud_service_type"`
+	AlternativeAHasUI                   null.String        `json:"alternativeAHasUI" db:"alternative_a_has_ui"`
+	AlternativeAPros                    null.String        `json:"alternativeAPros" db:"alternative_a_pros"`
+	AlternativeACons                    null.String        `json:"alternativeACons" db:"alternative_a_cons"`
+	AlternativeACostSavings             null.String        `json:"alternativeACostSavings" db:"alternative_a_cost_savings"`
+	AlternativeBTitle                   null.String        `json:"alternativeBTitle" db:"alternative_b_title"`
+	AlternativeBSummary                 null.String        `json:"alternativeBSummary" db:"alternative_b_summary"`
+	AlternativeBAcquisitionApproach     null.String        `json:"alternativeBAcquisitionApproach" db:"alternative_b_acquisition_approach"`
+	AlternativeBSecurityIsApproved      null.Bool          `json:"alternativeBSecurityIsApproved" db:"alternative_b_security_is_approved"`
+	AlternativeBSecurityIsBeingReviewed null.String        `json:"alternativeBSecurityIsBeingReviewed" db:"alternative_b_security_is_being_reviewed"`
+	AlternativeBHostingType             null.String        `json:"alternativeBHostingType" db:"alternative_b_hosting_type"`
+	AlternativeBHostingLocation         null.String        `json:"alternativeBHostingLocation" db:"alternative_b_hosting_location"`
+	AlternativeBHostingCloudServiceType null.String        `json:"alternativeBHostingCloudServiceType" db:"alternative_b_hosting_cloud_service_type"`
+	AlternativeBHasUI                   null.String        `json:"alternativeBHasUI" db:"alternative_b_has_ui"`
+	AlternativeBPros                    null.String        `json:"alternativeBPros" db:"alternative_b_pros"`
+	AlternativeBCons                    null.String        `json:"alternativeBCons" db:"alternative_b_cons"`
+	AlternativeBCostSavings             null.String        `json:"alternativeBCostSavings" db:"alternative_b_cost_savings"`
+	CreatedAt                           *time.Time         `json:"createdAt" db:"created_at"`
+	UpdatedAt                           *time.Time         `json:"updatedAt" db:"updated_at"`
+	ArchivedAt                          *time.Time         `db:"archived_at"`
+
+	// TODO: these fields are unused in GQL Schema but still exist as DB columns.
+	// Write a migration to drop these columns
+	// See https://jiraent.cms.gov/browse/EASI-1693
+	InitialSubmittedAt *time.Time `json:"initialSubmittedAt" db:"initial_submitted_at"`
+	LastSubmittedAt    *time.Time `json:"lastSubmittedAt" db:"last_submitted_at"`
+}
+
+// BusinessCaseWithCosts is a helper to allow for legacy REST code
+// to function that combines lifecycle costs with the business case model
+type BusinessCaseWithCosts struct {
+	BusinessCase
+	LifecycleCostLines EstimatedLifecycleCosts `json:"lifecycleCostLines" db:"lifecycle_cost_lines"`
```

Comment:
This new struct for legacy REST endpoints is the main change here alongside the new dataloaders.